scripts/oc-auth.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+export CLOUDIGRADE_BASE_URL="${CLOUDIGRADE_BASE_URL:-test.cloudigra.de}"
 export CLOUDIGRADE_USER=$(uuidgen --random)
```

Comment:
According to the README:

```
located in ``scripts/oc-auth.sh`` which will use ``CLOUDIGRADE_USER`` or create
unique name using ``uuidgen`` and create a superuser, and then retreive an
```

According to README, this line should reuse a previously set CLOUDIGRADE_USER env var, does'n it? If it should, then it should be updated to do what it is saying on README.

Reply:
updated readme